mods/ra/maps/soviet-13a/soviet13a.lua

```diff
@@ -0,0 +1,125 @@
+
+	Utils.Do(Jeeps, function(jeep)
+		jeep.Patrol(JeepWaypoints, true, 125)
+	end)
+
+	Utils.Do(StartAttack, function(a)
+		IdleHunt(a)
+	end)
+
+	ChronoCam = Actor.Create("camera", true, { Owner = USSR, Location = Chronosphere.Location})
+end
+
+MissionTriggers = function()
+	Trigger.OnAllKilledOrCaptured(RadarSites, function()
+		USSR.MarkCompletedObjective(TakeDownRadar)
+		ChronoshiftAlliedUnits()
+	end)
+
+	Trigger.OnCapture(Chronosphere, function()
+		if not USSR.IsObjectiveCompleted(TakeDownRadar) then
+			Media.DisplayMessage(UserInterface.Translate("chrono-trap-triggered"), UserInterface.Translate("headquarters"))
+			Chronosphere.Kill()
+		else
+			USSR.MarkCompletedObjective(CaptureChronosphere)
+		end
+	end)
+
+	Trigger.OnKilled(Chronosphere, function()
+		USSR.MarkFailedObjective(CaptureChronosphere)
+	end)
+
+	local chronoTriggered
+	Trigger.OnEnteredFootprint(ChronoDemolitionTrigger, function(actor, id)
+		if actor.Owner == USSR and not chronoTriggered and not USSR.IsObjectiveCompleted(TakeDownRadar) then
+			Trigger.RemoveFootprintTrigger(id)
+			chronoTriggered = true
+			Media.DisplayMessage(UserInterface.Translate("chrono-trap-triggered"), UserInterface.Translate("headquarters"))
+			Chronosphere.Kill()
+		end
+	end)
+
+	Trigger.OnEnteredProximityTrigger(ChinookLZ.CenterPosition, WDist.FromCells(5), function(actor, id)
+		if actor.Owner == USSR and actor.Type == "harv" then
+			Trigger.RemoveProximityTrigger(id)
+			SendChinook()
+		end
+	end)
+
+	local oreAttackTriggered
+	Trigger.OnEnteredFootprint(OreAttackTrigger, function(actor, id)
+		if actor.Owner == USSR and not oreAttackTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			oreAttackTriggered = true
+
+			Utils.Do(OreAttackers, function(a)
+				if not a.IsDead then
+					IdleHunt(a)
+				end
+			end)
+		end
+	end)
+end
+
+ChronoshiftAlliedUnits = function()
+	if Chronosphere.IsDead then
+		return
+	end
+
+	local cells = Utils.ExpandFootprint({ ChronoshiftPoint.Location }, false)
+	local units = { }
+	for i = 1, #cells do
+		local unit = Actor.Create("2tnk", true, { Owner = Greece, Facing = Angle.North })
+		units[unit] = cells[i]
+		IdleHunt(unit)
+	end
+	Chronosphere.Chronoshift(units)
+end
+
+Tick = function()
+	Greece.Cash = 20000
+
+	if USSR.HasNoRequiredUnits() then
+		Greece.MarkCompletedObjective(AlliesObjective)
+	end
+end
+
+WorldLoaded = function()
+	USSR = Player.GetPlayer("USSR")
+	Greece = Player.GetPlayer("Greece")
+	GoodGuy = Player.GetPlayer("GoodGuy")
+
+	InitObjectives(USSR)
+
+	AlliesObjective = AddPrimaryObjective(Greece, "")
+	TakeDownRadar = AddPrimaryObjective(USSR, "destroy-allied-radar-sites")
+	CaptureChronosphere = AddPrimaryObjective(USSR, "capture-the-chronosphere")
+
+	Camera.Position = DefaultCameraPosition.CenterPosition
+	Start()
+	MissionTriggers()
+	ActivateAI()
+	GreeceWarFactory.RallyPoint = WarFactoryRally.Location
+	if Map.LobbyOption("difficulty") == "hard" then
```

Comment:
Can we use the global variable here?
```suggestion
	if Difficulty == "hard" then
```

Reply:
force pushed